Update documentation about the environment file (#27)
scalar-labs/scalar-terraform-examples: azure/README.md

$ terraform apply
```

> In the output of `terraform apply` command, the `cosmosdb_account_endpoint` and the `cosmosdb_account_primary_master_key` are used to configure Scalar DL in the next section. Please take a note of them.

Reviewer: The sentence sounds a bit weird. You mean this?
```suggestion
In the output of `terraform apply` command, you will see  `cosmosdb_account_endpoint` and `cosmosdb_account_primary_master_key`. Please take note of them since they are used to configure Scalar DL in the next section. 
```

### Create Scalar DL resources

Before running the terraform command to create Scalar DL resources, you need to prepare an environment file for the container configuration which will be loaded by `docker-compose`.

> The default file name is `scalardl_conatainer.env`. Please copy the example file to that name.

Reviewer 2: ```suggestion
The default file name is `scalardl_container.env`. Please copy the example file to that name.
```

Author: Thanks!

```console
$ cd azure/scalardl
$ cp example.scalardl_container.env scalardl_container.env
```

Then, update the contents of the file.

* Cassandra

    To use Cassandra, you can keep the contents as long as you use the default settings.

    ```
    SCALAR_DB_STORAGE=cassandra
    SCALAR_DB_CONTACT_POINTS=cassandra-lb.internal.scalar-labs.com
    SCALAR_DB_CONTACT_PORT=9042
    SCALAR_DB_USERNAME=cassandra
    SCALAR_DB_PASSWORD=cassandra
    ```

* Cosmos DB

    To use Cosmos DB, you should find the URL of the account endpoint and the master key in the output of the `cosmosdb` module. You can get them at any time by running `terraform output` in the `cosmosdb` directory. Please specify the endpoint URL as `SCALAR_DB_CONTACT_POINTS` and the master key as `SCALAR_DB_PASSWORD`.

>     `SCALAR_DB_STORAGE` must be `cosmos`. `SCALAR_DB_CONTACT_PORT` and `SCALAR_DB_USERNAME` are not used for Cosmos DB, so you can keep them blank.

Reviewer: ```suggestion
    `SCALAR_DB_STORAGE` must be `cosmos`. `SCALAR_DB_CONTACT_PORT` and `SCALAR_DB_USERNAME` are not used for Cosmos DB, so you can leave them blank.
```
